civihr_default_theme/scss/components/_panel.scss

```diff
@@ -152,11 +152,13 @@ form {
     padding: 0;
     position: absolute;
     right: 20px;
-    top: 17px;
+    top: 13px;
 
     > [class*="chr_panel__toolbar__"] {
       display: inline-block;
+      line-height: 34px;
```

Comment:
I can notice 4 places in this code change where the number `34` is used. I wonder if it is a coincidence, and if it is not - the number should be stored in a variable.

Reply:
Used variable `$input-height-base`.